src/interfaces/alteration-interface.ts

```diff
@@ -1,6 +1,6 @@
 import {
   BusinessIF, NameRequestIF, NameTranslationIF, ShareStructureIF, ContactPointIF
-} from '@/interfaces'
+} from './'
```

Comment:
No, should probably point to
`@bcrs-shared-components/interfaces`.

Reply:
Otherwise THIS file assumes the other files are in the local folder.

Reply:
that was my initial thought/attempt as well, though it threw TS errors in the Shared Components. Will have a closer look.

Reply:
Ah, ok it was throwing TS errors: 'cannot find module @bcrs-shared-components/interfaces' because..

This directory has it's own node_modules, where it is looking for this..

![Screen Shot 2021-02-25 at 9 38 24 AM](https://user-images.githubusercontent.com/53542131/109193447-375d9880-774d-11eb-8981-4d562531f179.png)

So, the only option i see is to add the dependency (on itself!?) in the package.json

![Screen Shot 2021-02-25 at 9 41 12 AM](https://user-images.githubusercontent.com/53542131/109193810-9cb18980-774d-11eb-8f86-67578af12956.png)

Reply:
😆  what do you think we should do?

Reply:
If you think this is ok, i will commit this.

Reply:
Yeah that doesn't feel right.
Maybe this was correct after all: `import { blah} from './'`.

Reply:
It means that project that import these interfaces will need to import all dependent interfaces too (eg, in index.ts file). I _think_ that's OK, not 100% sure though.

Reply:
kk, copy that! This would be g2g then